lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py

```diff
@@ -200,7 +202,6 @@ def main():
     argument_spec = cs_argument_spec()
     argument_spec.update(dict(
         name=dict(required=True),
-        affinty_type=dict(removed_in_version='2.9'),
```

Comment:
I would prefer to move that into a separate PR. Feature removal should also have a changelog (`removed_features:`) and potentially also a porting guide entry (no idea where though).

Reply:
this deprecation was due to a typo in a param name and is not noteworthy becuase it should be clear to use the one without a typo...